dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentCodeGenerator.java

```diff
@@ -0,0 +1,97 @@
+package com.dotcms.experiments.business;
+
+import static com.dotmarketing.util.FileUtil.getFileContentFromResourceContext;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Logger;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Util class to generate the JS/HTML code needed to Support Experiment into the Browser.
+ */
+public enum ExperimentCodeGenerator {
```

Comment:
I think this should be on web or view package, since mostly printing js

Reply:
done https://github.com/dotCMS/core/pull/24321/commits/3dfbcf6cd202e85cfcdf8cd509c2990d16e05b48#diff-cbb6bb76d043d2cb96b6503fafe6b5b1e5525c9b6fb58844a57a397b1f037cb6R57